leo/commands/package/install.rs

```diff
@@ -29,31 +30,65 @@ use tracing::span::Span;
 pub struct Install {}
```

Comment:
Per the discussion, we should re-name this command `fetch` as install usually means installing a binary to the path rather than loading the dependencies.

Reply:
Just changed it to fetch. 👍